### ISSUE TITLE
Issues with retrieval

### ISSUE THREAD

**Issue by quantixed** [nicholasmfraser/rbiorxiv#2]
I am using the development version of rbiorxiv and have run into an issue.
If I try to retrieve data for the first two months of 2021
`data <- biorxiv_content(from = "2021-02-01", to = "2021-02-28", limit = "*", format = "df")`
I get `Error: no posts found`. Using an equivalent URL in a browser retrieves the records and the message is status OK.
After some investigation, I think a preprint from 2021-01-08 is to blame.
`data <- biorxiv_content(from = "2021-01-08", to = "2021-01-08", limit = "*", format = "df")`
gives the error but other days do not, and then
`data <- biorxiv_content(from = "2021-01-08", to = "2021-01-08", limit = "*", format = "df", skip = 11)`
gives the error but
`data <- biorxiv_content(from = "2021-01-08", to = "2021-01-08", limit = "*", format = "df", skip = 12)`
does not.
I am wondering if the parser is choking on this particular record?
I _think_ it is this one 10.1101/2020.05.08.084327 but I am not certain. I can't see anything weird about it.

**Comment by quantixed** [comment]
A bit more information to add. The record mentioned above it _not_ the problem. I believe the issue is the endpoint used in the code, i.e. it is requesting records that don't exist.
The API says that there are a total of 212 records for this date range, but if we query by setting cursor to 190
https://api.biorxiv.org/details/biorxiv/2021-01-08/2021-01-08/190 there is only 1 record retrieved. 
Setting it to 191 gives `{"messages":[{"status":"no posts found"}], "collection":[]}`
I don't know why the total number of records is incorrect, but I guess a fix would be to check that `length(data)` grows by `max_results_per_page` with each iteration (around line 87 in https://github.com/nicholasmfraser/rbiorxiv/blob/master/R/biorxiv_content.R) and break if not?

**Comment by nicholasmfraser** [comment]
Hi @quantixed! Thanks for the report and example which I was able to reproduce. This is odd behaviour - the bioRxiv API is indicating that more results will be returned than actually are, I do not know why. 

I've just issued a fix (https://github.com/nicholasmfraser/rbiorxiv/commit/b5f5ed7d7fd1730fb822a555b91ad81faa06a335) which should take care of this - the examples you provided above are all working. So far I only added this to the `biorxiv_content()` endpoint, and not the `biorxiv_published()` or `biorxiv_publisher()` endpoints - I plan to do some more checks and tests in the coming days in case there are some edge cases I am not covering. If you find some more abnormalities then let me know.

NB the CRAN version of this package is currently unavailable but should be restored shortly. For now I would recommend using the development version (as you already seem to be). You might also find the [`medrxiv` ](https://github.com/ropensci/medrxivr) package useful, depending on your needs.

p.s. unrelated, but I enjoyed your blog post last year on [bioRxiv screening times](https://quantixed.org/2020/03/30/screenager-screening-times-at-biorxiv/).
